### PR TITLE
regen with the latest gir

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: src/auto/event.rs [hunk 1]
```diff
@@ -1,162 +0,0 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir)
```
Comment (sfanxiang) [inline src/auto/event.rs:1]: Not sure why `event.rs` gets deleted, but it doesn't look right.

Any ideas? cc @alatiera @GuillaumeGomez

Comment (alatiera) [reply src/auto/event.rs:1]: There'es api missing from auto/surface and auto/seat as well, need to look into

**Comment by alatiera** [comment]
What was the gir you regened against?

**Comment by bilelmoussaoui** [comment]
> What was the gir you regened against?

The one in gir-files repository

**Pull request closed without merging**